core/assets/bundles/bundle_fr.properties

```
@@ -2330,7 +2330,7 @@ lcategory.block.description = Interagit avec les blocs.
 lcategory.operation = Opérations
 lcategory.operation.description = Opérations logiques.
 lcategory.control = Contrôle des Flux
-lcategory.control.description = Manipule le flot d'exécution.
+lcategory.control.description = Manipule les ordres d'exécution.
```

```suggestion
lcategory.control.description = Manipule l'ordre d'exécution.
```
Il n'y a pas plusieurs programmes qui s'éxecutent en même temps dans un même processeur.